### PR TITLE
feat(db) : create migrations to update column names and add foreign key constraint

### PR DESCRIPTION
**Pull request by Sukhacoder02**
- change the column names of the table to snake_case
- create a migration to add foreign key constraint to case_studies table

### CODE REVIEW

**Comment by rohini-senthil** [comment]
Please name your PR titles as you would do with commits

File: database/migrations/20230207133247-create-users.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
 module.exports = {
   async up(queryInterface, Sequelize) {
     await queryInterface.createTable('users', {
-
```
Comment (rohini-senthil) [inline database/migrations/20230207133247-create-users.js:6]: Can we create new migrations with column option `field` (https://sequelize.org/docs/v6/core-concepts/model-basics/#column-options). We are not supposed to edit existing migrations

Comment (Sukhacoder02) [reply database/migrations/20230207133247-create-users.js:6]: I have made the changes as directed.

File: database/migrations/20230209024110-update-column-names-for-users.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    queryInterface.renameColumn('users', 'userId', 'user_id');
+    queryInterface.renameColumn('users', 'currentEngagementIds', 'current_engagement_ids');
+    queryInterface.renameColumn('users', 'caseStudies', 'case_studies');
```
Comment (rohini-senthil) [inline database/migrations/20230209024110-update-column-names-for-users.js:14]: Can we make this `case_study_ids`?

Comment (Sukhacoder02) [reply database/migrations/20230209024110-update-column-names-for-users.js:14]: Sure, added a migration to  change ```case_studies``` to  ```case_study_id``` in users table and updated the ```users``` model accordingly.

Comment (rohini-senthil) [reply database/migrations/20230209024110-update-column-names-for-users.js:14]: But I think it's fine.

**Review by rohini-senthil — APPROVED** [review]

**Pull request merged**